## HACK indeed, let's not rpath the world, mm-kay?  -- rex 

diff -up sip-4.13.3/siputils.py.no_rpath sip-4.13.3/siputils.py
--- sip-4.13.3/siputils.py.no_rpath	2012-06-22 06:54:54.311308656 -0500
+++ sip-4.13.3/siputils.py	2012-06-22 06:55:58.998499957 -0500
@@ -434,7 +434,7 @@ class Makefile:
             if l_dir in ("", ".", ".."):
                 continue
 
-            rpaths.append(l)
+            #rpaths.append(l)
 
         if self._python:
             incdir.append(self.config.py_inc_dir)
@@ -603,7 +603,7 @@ class Makefile:
             # Handle library directories.
             libdir_qt = self.optional_list("LIBDIR_QT")
             libdir.extend(libdir_qt)
-            rpaths.extend(libdir_qt)
+            #rpaths.extend(libdir_qt)
 
             if qt_version >= 0x040000:
                 # Try and read QT_LIBINFIX from qconfig.pri.
